src/components/Header/_header.scss

```diff
@@ -47,3 +47,37 @@
     display: flex;
   }
 }
+
+.#{$prefix}--header__menu-item[role='menuitem'] {
+  display: flex;
+  align-items: center;
+  color: #c6c6c6;
+  height: 100%;
+  font-size: 0.875rem;
+  font-weight: 400;
+  letter-spacing: 0;
+  line-height: 1.125rem;
+  text-decoration: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
```

Comment:
shouldn't autoprefixer take care of these prefixes for us?

Reply:
yes good catch. had to copy some carbon styles